scripts/pre_push_hook.py

```diff
@@ -295,21 +295,20 @@ def _install_hook():
         print 'Copied file to .git/hooks directory'
 
 
-def does_diff_include_js_files(files_to_lint):
-    """Returns true if diff includes JS files.
+def does_diff_include_frontend_scripts(files_to_lint):
```

Comment:
Can we change `scripts` to `files` here and below just to maintain uniformity?

Reply:
.html is also a frontend file, right?

Reply:
Hmm -- I agree. How about `does_diff_include_js_and_ts_files`?

Reply:
I think it should be or instead of and, right?

Alos, ts_files = typeScript_file so can we make does_diff_include_script_files (not sure!).

Reply:
One more reason for having `does_diff_include_frontend_scripts`
```
if does_diff_include_frontend_scripts(..):
    run frontend_tests
```

This makes it kinda more readable.

But I'm fine with renaming, should I make it does_diff_include_js_or_ts_files?

Reply:
Yes, it should be `does_diff_include_js_or_ts_files`. I am not sure about the latter name though. I think we can go with the former. What do you suggest? Thanks!